src/utils/error/code.js

```diff
@@ -0,0 +1,676 @@
+//@flow
+type CodeType = { [code: string]: number };
+
+const Code: CodeType = {
+  /**
+   * A network request was made using an unsupported URI scheme.
+   * <br> error.data[0] is the URI.
```

Comment:
remove the shaka docs which are not related to the specific explanation of the code.
i.e <br>...